proteobench/modules/dda_quant/module.py

```diff
@@ -343,6 +359,8 @@ def clone_pr(
         current_datapoint = temporary_datapoints.iloc[-1]
         current_datapoint["is_temporary"] = False
         for k, v in datapoint_params.__dict__.items():
+            if k == "software_name":
```

Comment:
This needs to be removed?

Reply:
Indeed, fixed in a (hopefully) more elegant way.